app/components/products/productsService.js

```diff
@@ -0,0 +1,7 @@
+eShopApp.factory('products', function ($http) {
+    return {
+        getProducts: function () {
+            return $http({method: 'GET', url: 'https://localhost:44302/api/Products'});
```

Comment:
the api reference still constant

Reply:
it's changed but I don't know why it appears like that for you

Reply:
yes i checked it in the branch itself and it's changed 👍